src/marin/evaluation/evaluators/vllm_tpu_evaluator.py

```diff
@@ -121,37 +120,18 @@ def cleanup(model: ModelConfig, vllm_port: int | None = None) -> None:
         model.destroy()
 
     _python_version: str = "3.10"
-    _pip_packages: ClassVar[list[Dependency]] = DEFAULT_PIP_PACKAGES
-    _py_modules: ClassVar[list[Dependency]] = []
+    _extras: ClassVar[list[Dependency]] = []
```

Comment:
initializing this statically as `[]` is bad yeah?

Reply:
I thought so as well, but the previous code was doing this (just with the DEFAULT_PIP_PACKAGES = []). i can switch to a None sentinel instead.